feature/PB-2 create navbar

Cacatuaa opened the pull request.

Cacatuaa committed:
feature/PB-2-create-navbar

Criação inicial da navbar com alguns pontos pendentes, sendo eles:
- Botão de Light/Dark mode;
- Estilizar o texto do título e dos botões;

Cacatuaa committed:
feature/PB-2-create-navbar

Criação inicial da navbar com alguns pontos pendentes, sendo eles:
- Botão de Light/Dark mode;
- Estilizar o texto do título e dos botões;

Jejugo commented on src/app/page.js:
```diff
@@ -1,11 +1,15 @@
 import styled from "styled-components";
+import Navbar from "../components/Navbar";
 
 const Root = styled.div`
```
Now that you know the concept of `layout.js` you can probably use it instead of using this style on `page.js`

Jejugo commented on src/components/Navbar/index.jsx:
```diff
@@ -0,0 +1,86 @@
+import Image from "next/image";
+import Link from "next/link";
+
+const Main = styled.div`
```
Try separating this in a separate file. Import it as `import * as S from "./styles"`. Do it for every styled-component below

Jejugo replied:
Also, I now added a theme to our project. Try grabbing values such as fontsizes, colors, spacing and radius directly from the theme

Jejugo reviewed (dismissed):
Nice job man!

Cacatuaa committed:
Chore/PB-2: Adjusting Navbar component

Adjusting navbar component with the following changes:
- Exporting the styled components to a separeted file;

Cacatuaa committed:
Chore/PB-2 Applying prettier

Cacatuaa committed:
Chore/PB-2 Adjusting Navbar to layout instead page
